doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
In C parser verbs are hardcoded into state machine, so we didn't really have a choice.

But the new parser should support any valid verbs imho, it's not its job to enforce any limitations like this.

PS: I want BREW method, 'cause hey it's RFC :P